### PR TITLE
convert datetime.min to 1970-01-01

### PR DESCRIPTION
**Pull request by snopoke**
http://manage.dimagi.com/default.asp?167722

@sravfeyn @dannyroberts

### CODE REVIEW

File: couchexport/models.py [hunk 1]
```diff
@@ -113,18 +113,10 @@ def __unicode__(self):
...
     @classmethod
     def wrap(cls, data):
-        ret = super(ExportSchema, cls).wrap(data)
-        if not ret.timestamp:
-            # this is a very old export. notify that something weird happened,
-            # but just start from the beginning of time.
-            notify_exception(
-                None,
-                'an export without a timestamp was accessed! %s (%s)' % (ret.index, ret._id)
-            )
```
Comment (snopoke) [inline couchexport/models.py]: The last time this happened was 2014-12-22

Comment (czue) [inline couchexport/models.py]: nice.

**Comment by sravfeyn** [comment]
Kicking CI again

**Comment by czue** [comment]
build fail isn't related (looks like dependency changes). @snopoke do you want this merged or should we fix build?

```
Traceback (most recent call last):
  File "/home/travis/virtualenv/python2.7.9/bin/django-admin.py", line 5, in <module>
    management.execute_from_command_line()
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/django/core/management/__init__.py", line 399, in execute_from_command_line
    utility.execute()
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/django/core/management/__init__.py", line 392, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/django/core/management/commands/test.py", line 50, in run_from_argv
    super(Command, self).run_from_argv(argv)
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/django/core/management/base.py", line 242, in run_from_argv
    self.execute(*args, **options.__dict__)
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/django/core/management/commands/test.py", line 71, in execute
    super(Command, self).execute(*args, **options)
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/django/core/management/base.py", line 285, in execute
    output = self.handle(*args, **options)
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/django/core/management/commands/test.py", line 88, in handle
    failures = test_runner.run_tests(test_labels)
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/django/test/runner.py", line 144, in run_tests
    suite = self.build_suite(test_labels, extra_tests)
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/django/test/simple.py", line 243, in build_suite
    app = get_app(label)
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/django/db/models/loading.py", line 179, in get_app
    self._populate()
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/django/db/models/loading.py", line 78, in _populate
    self.load_app(app_name)
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/django/db/models/loading.py", line 99, in load_app
    models = import_module('%s.models' % app_name)
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/django/utils/importlib.py", line 40, in import_module
    __import__(name)
  File "/home/travis/build/dimagi/couchexport/couchexport/models.py", line 8, in <module>
    from dimagi.ext.couchdbkit import Document, DictProperty,\
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/dimagi/__init__.py", line 2, in <module>
    import test_utils
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/dimagi/test_utils/__init__.py", line 3, in <module>
    from .base import *
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/dimagi/test_utils/base.py", line 15, in <module>
    from dimagi.utils.couch.loosechange import AssocArray
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/dimagi/utils/couch/__init__.py", line 6, in <module>
    from dimagi.utils.couch.cache.cache_core import get_redis_client
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/dimagi/utils/couch/cache/cache_core/__init__.py", line 73, in <module>
    from .api import *
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/dimagi/utils/couch/cache/cache_core/api.py", line 2, in <module>
    from redis_cache.exceptions import ConnectionInterrumped
ImportError: No module named redis_cache.exceptions
```

**Comment by snopoke** [comment]
@czue just made a commit that should fix the build

**Pull request merged**